pulp_container/app/registry.py

```diff
@@ -76,6 +78,10 @@ async def _dispatch(file, headers):
         file_response = web.FileResponse(path, headers=full_headers)
         return file_response
 
+    @RegistryContentCache(
+        base_key=lambda req, cac: Registry.find_base_path_cached(req, cac),
+        auth=lambda req, cac, bk: Registry.auth_cached(req, cac, bk),
+    )
```

Comment:
I think these should error since it should try to await the auth and base_key methods which are non-async.